lib/go/edgecontext/backend.go

```diff
@@ -15,12 +14,24 @@ func init() {
 
 var backend atomic.Pointer[Backend]
 
+// Backend provides the interface for the edgecontext package to interact with the underlying implementation.
 type Backend struct {
-	Set        func(context.Context, *EdgeRequestContext) context.Context
-	Get        func(context.Context) (*EdgeRequestContext, bool)
-	Factory    func(Config) ecinterface.Factory
-	Init       func(Config) *Impl
-	New        func(context.Context, *Impl, NewArgs) (*EdgeRequestContext, error)
+	// Set is the implementation for SetEdgeContext.
```

Comment:
🔕 should this explicitly say like, `ecinterface.Interface.SetEdgeContext` to be explicit?

Reply:
Seeing as that's not even what it is... yes it should 😅